exotica_core/include/exotica_core/problems/dynamic_time_indexed_shooting_problem.h

```diff
@@ -30,6 +30,7 @@
 #ifndef EXOTICA_CORE_DYNAMIC_TIME_INDEXED_SHOOTING_PROBLEM_H_
 #define EXOTICA_CORE_DYNAMIC_TIME_INDEXED_SHOOTING_PROBLEM_H_
 
+#include <exotica_core/dynamics_solver.h>
```

Comment:
Unnecessary include

Reply:
True. Was needed by ilqr_solver so I put it there instead.

Reply:
Still here?